dff/pipeline/types.py

```diff
@@ -171,15 +176,6 @@ class ExtraHandlerType(Enum):
 ]
 
```

Comment:
Add docstring for `ExtraHandlerBuilder`.

Reply:
What should we write there?